apps/run.go

```diff
@@ -254,7 +254,13 @@ func (ctx *runContext) exitCode() (int, error) {
 	if err != nil {
 		return -1, errgo.Mask(err, errgo.Any)
 	}
-	req.SetBasicAuth("", config.AuthenticatedUser.AuthenticationToken)
+
+	token, err := config.C.TokenGenerator.GetAccessToken()
+	if err != nil {
+		return -1, errgo.Notef(err, "fail to generate auth")
+	}
+
+	req.SetBasicAuth("", token)
```

Comment:
Does this work?
The JWT are use with Bearer auth not Basic auth.

Reply:
```
└> ./scalingo -a db run bash
-----> Connecting to container [one-off-1598]...  
-----> Process 'bash' is starting...  

   _____               __ _                    
  / ___/ _____ ____ _ / /(_)____   ____ _ ____ 
  \__ \ / ___// __ `// // // __ \ / __ `// __ \
 ___/ // /__ / /_/ // // // / / // /_/ // /_/ /
/____/ \___/ \__,_//_//_//_/ /_/ \__, / \____/ 
                                /____/         

```

Reply:
Authentication too permissive?

Reply:
Oh I know actually

Reply:
The token is sent to the run service (`one-off.scalingo.com`) which still expect basic auth